src/main/java/mekanism/common/recipe/serializer/MekanismRecipeSerializer.java

```diff
@@ -265,4 +266,17 @@ public static <RECIPE extends ItemStackChemicalToItemStackRecipe & IBasicItemSta
               factory
         ));
     }
+
+    public static MekanismRecipeSerializer<BasicPhaseShiftingRecipe> phaseShifting(Function3<ChemicalStackIngredient, ChemicalStack, Long, BasicPhaseShiftingRecipe> factory) {
```

Comment:
This factory and return generic is wrong - it should follow the others and have a `RECIPE extends` generic